modules/displaydynamic.js

```diff
@@ -0,0 +1,36 @@
+export default class displayDynamic {
+  static displayBooks() {
+    const books = Storage.getBooks();
+    books.forEach((book) => displayDynamic.addBooksCollection(book));
+  }
```

Comment:
- Good job for using normal function here :+1: But for best practice following the ES6 syntaxes, you should use arrow syntax for your functions. Your functions should look like this: `displayBooks = () => {...}`. Kindly address that. :grin: